configs/example.ini

```diff
@@ -1,13 +1,13 @@
 [main]
-; Twitch login name of the bot account
-nickname = the_bot_name
-; Twitch login name of the channel the bot runs in
-streamer = streamer_name
-; login name of the primary admin (will be granted level 2000 initially)
-admin = your_own_name
-; an additional channel the bot will join and receive commands from. (login name)
+; Twitch User ID of the bot account
+bot_id = 123456789
+; Twitch User ID of the channel the bot runs in
+streamer_id = 123456789
+; Twitch User ID of the primary admin (will be granted level 2000 initially)
+admin_id = 123456789
+; Twitch User ID of an additional channel the bot will join and receive commands from.
 ; You can remove this config entry if you don't want to use this feature
-control_hub = a_channel_for_all_your_bots_to_join
+control_hub_id = 123456789
```

Comment:
Further down in the config there is a `streamer_name` option under `[web]`, shouldn't that be removed/replaced as well?